website/content/en/preview/concepts/instance-types.md

```diff
@@ -9509,8 +9513,9 @@ below are the resources available with some assumptions and after the instance o
  |--|--|
  |cpu|127610m|
  |ephemeral-storage|17Gi|
- |memory|477043Mi|
- |pods|688|
+ |memory|480816Mi|
+ |pods|345|
+ |vpc.amazonaws.com/pod-eni|108|
```

Comment:
eh?

Reply:
This is just the output of running `make docgen`. I could add it in a separate PR but it validates that the `docgen` is performing like it should